### PR TITLE
fix(scripts): check the correct variable in check-engines-cli.sh

### PR DESCRIPTION
**Pull request by aqrln**
Fixes

```
../../.github/scripts/check-engines-cli.sh: line 104: [: ==: unary operator expected
```

### CODE REVIEW

**Comment by aqrln** [comment]
Adding `set -e` uncovered another issue

```
--- yarn prisma -v ---
Unknown Syntax Error: Command not found; did you mean one of:

  0. yarn cache clean [--mirror] [--all]
  1. yarn cache clean [--mirror] [--all]
  2. ....
```

This is something to be fixed separately, I'll remove `set -e` back for now.

**Comment by aqrln** [comment]
https://github.com/prisma/e2e-tests/issues/2251

File: .github/scripts/check-engines-cli.sh [hunk 1]
```diff
@@ -5,17 +5,25 @@ echo "-------------- Checking CLI/Engines QE Engine --------------"
 DIR=$1
 PROJECT=$2
 
-DEFAULT_CLI_QUERY_ENGINE_TYPE='library'
+DEFAULT_QUERY_ENGINE_TYPE='library'
 
 # Check to see if the env var "PRISMA_CLI_QUERY_ENGINE_TYPE" is set if not then using the default
```
Comment (Jolg42) [inline .github/scripts/check-engines-cli.sh]:
```suggestion
# Check to see if the env var "DEFAULT_QUERY_ENGINE_TYPE" is set if not then using the default
```

Comment (janpio) [reply .github/scripts/check-engines-cli.sh]: But the code checks for `PRISMA_CLI_QUERY_ENGINE_TYPE`.

**Comment by Jolg42** [comment]
It's confusing to me that there is "Client" logic in this file named `check-engines-cli.sh` and that we also have a file called `check-engines-client.sh` where it would make more sense to me https://github.com/prisma/e2e-tests/blob/aqrln/fix-check-engines-cli-script/.github/scripts/check-engines-client.sh

**Comment by aqrln** [comment]
Also it seems that this logic is actually present in `check-engines-client.sh`. ~~The dataproxy check should probably just be removed from `check-engines-cli.sh`.~~

**Comment by aqrln** [comment]
Oh, I see. Given these lines:

https://github.com/prisma/e2e-tests/blob/c2c421098204b12191d95db9900716136bc8e6e1/.github/scripts/check-engines-cli.sh#L94-L95

It looks like it probably was a typo and `$CLI_QUERY_ENGINE_TYPE` was meant to be checked instead of `$CLIENT_ENGINE_TYPE`.

**Review by Jolg42 — APPROVED** [review]
Nice!

**Pull request merged**